flexx/ui/widgets/_markdown.py

```diff
@@ -0,0 +1,44 @@
+"""
+
+Simple example:
+
+.. UIExample:: 200
+
+    # Define data. This can also be generated with the plotly Python library
+    data = [{'type': 'bar',
```

Comment:
This example looks to be from another file :)

Reply:
Added relevent comments